lib/build/bundle.js

```diff
@@ -220,15 +238,21 @@ exports.Bundle = class {
       if (includeBundles) {
         bundleMetadata[currentBundle.moduleId] = currentBundle.getBundledModuleIds();
       }
-
-      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId;
+      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId + (buildOptions.rev && currentBundle.hash ? '-' + currentBundle.hash : '');
     }
 
     if (includeBundles) {
       config.bundles = bundleMetadata;
     }
 
-    return 'requirejs.config(' + JSON.stringify(config) + ');'
+    return 'function _loadAureliaRequireJSConfig(){requirejs.config(' + JSON.stringify(config) + ');}'
+  }
+  
+  setIndexFileConfigTarget(platform, location){
```

Comment:
Can add some code comments that explain a bit of what is going on there. I'm pretty sure I've got the idea but I might have trouble looking back on it in another month. So, perhaps some comments on this method and a few other places throughout the code above to explain the process.